docs/CONNECTOR_PROTOCOL.md

```diff
@@ -82,29 +110,111 @@ This is our main communication index, used to communicate the connector's config
 #### Elasticsearch mappings for `.elastic-connectors`:
 ```
 "mappings" : {
+  "_meta" : {
+    "pipeline" : {
+      "default_extract_binary_content" : true,
+      "default_name" : "ent-search-generic-ingestion",
+      "default_reduce_whitespace" : true,
+      "default_run_ml_inference" : true
+    },
+    "version" : "1"
+  },
   "properties" : {
     "api_key_id" : { "type" : "keyword" },
     "configuration" : { "type" : "object" },
     "description" : { "type" : "text" },
-    "error" : { "type" : "text" },
+    "error" : { "type" : "keyword" },
```

Comment:
I think we probably should change [here](https://github.com/elastic/ent-search/blob/e847748e7bb192787985caa7bc5d33bb2e068856/shared_togo/lib/shared_togo/crawler2/connector.rb#L32) instead of in the doc.

Reply:
We could, but then we'd need to make a migration to reindex data for all our 8.4 and 8.5 customers. And since it's not really critical that we be able to execute text search queries on this field right now, I don't think this is the time to do that.

Reply:
Does [this](https://github.com/elastic/ent-search/pull/6984#discussion_r1010023401) make sense?

I agree this is not critical to change now.